GLAA.Services/Automapper/PublicRegisterProfile.cs

```diff
@@ -0,0 +1,24 @@
+﻿using AutoMapper;
+using GLAA.Domain.Models;
+using GLAA.Services.PublicRegister;
+using GLAA.ViewModels.PublicRegister;
+
+namespace GLAA.Services.Automapper
+{
+    public class PublicRegisterProfile : Profile
+    {
+        public PublicRegisterProfile()
+        {
+            CreateMap<Licence, PublicRegisterLicenceSummaryViewModel>()
+                .ForMember(x => x.Id, opt => opt.MapFrom(y => y.Id))
+                .ForMember(x => x.ApplicationId, opt => opt.MapFrom(y => y.ApplicationId))
+                .ForMember(x => x.MostRecentStatus, opt => opt.MapFrom(y => StatusProfile.MapLicenceStatusViewModel(PublicRegisterViewModelBuilder.GetLatestStatus(y))))
+                .ForMember(x => x.BusinessName, opt => opt.MapFrom(y => y.BusinessName))
+                .ForMember(x => x.TradingName, opt => opt.MapFrom(y => y.TradingName))
+                .ForMember(x => x.Country, opt => opt.MapFrom(y => y.Address.Country))
+                .ForMember(x => x.County, opt => opt.MapFrom(y => y.Address.County))
+                .ForMember(x => x.IsApplication, opt => opt.MapFrom(y => PublicRegisterViewModelBuilder.GetLatestStatus(y).Status.IsApplication))
```

Comment:
I think this logic is in danger of getting out of hand. I don't like DB queries within the mapping. I think ideally we should have an `IsApplication` field on the licence so the mapping becomes simpler. It's a more major change but I can see this needed in a lot of places

Reply:
Should we have another task for this to add this field and change the references?  Or should I shuffle it into this bit of work?